l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -1246,16 +1246,17 @@ def importFatturaPA(self, cr, uid, ids, context=None):
                 raise orm.except_orm(
                     _("Error"), _("File is linked to invoices yet"))
             # decrypt  p7m file
-            if fatturapa_attachment.datas_fname.endswith('.p7m'):
-                temp_file_name = '/tmp/%s' % fatturapa_attachment.datas_fname
+            if fatturapa_attachment.datas_fname.lower().endswith('.p7m'):
+                temp_file_name = (
+                    '/tmp/%s' % fatturapa_attachment.datas_fname.lower())
                 temp_der_file_name = (
-                    '/tmp/%s_tmp' % fatturapa_attachment.datas_fname)
+                    '/tmp/%s_tmp' % fatturapa_attachment.datas_fname.lower())
                 with open(temp_file_name, 'w') as p7m_file:
                     p7m_file.write(fatturapa_attachment.datas.decode('base64'))
                 xml_file_name = os.path.splitext(temp_file_name)[0]
 
                 # check if temp_file_name is a PEM file
-                file_is_pem = self.check_file_is_pem(temp_file_name)
+                file_is_pem = self.check_file_is_pem(temp_file_name.lower())
```

Comment:
allora mi e' sfuggito...

2015-06-05 17:24 GMT+02:00 Lorenzo Battistini <notifications@github.com>:

> In l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py
> <https://github.com/eLBati/l10n-italy/pull/62#discussion_r31822071>:
>
> >                  with open(temp_file_name, 'w') as p7m_file:
> >                      p7m_file.write(fatturapa_attachment.datas.decode('base64'))
> >                  xml_file_name = os.path.splitext(temp_file_name)[0]
> >
> >                  # check if temp_file_name is a PEM file
> > -                file_is_pem = self.check_file_is_pem(temp_file_name)
> > +                file_is_pem = self.check_file_is_pem(temp_file_name.lower())
>
> Questo a cosa serve? temp_file_name è già lower case
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eLBati/l10n-italy/pull/62/files#r31822071>.
>



-- 
Best Regards
Alessio Gerace


*   Think Studio*
     +39 3396911693
     www.thinkstudio.it